### PR TITLE
Edit SAKE token address

### PR DESCRIPTION
**Pull request by therealsakeswap**
update sake token address

### CODE REVIEW

File: spaces/sake/index.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "symbol": "SAKE",
   "name": "Sake Swap",
   "defaultView": "core",
-  "address": "0x066798d9ef0833ccc719076Dab77199eCbd178b0",
-  "token": "0x066798d9ef0833ccc719076Dab77199eCbd178b0",
+  "address": "0xd8EA0752e500fb46c2F65C31B7B046B28aF09CE9",
+  "token": "0xd8EA0752e500fb46c2F65C31B7B046B28aF09CE9",
```
Comment (e-asia) [inline spaces/sake/index.json:7]: Something is wrong with the token, the call to query decimals() is failing. Is this a standard ERC20 token?

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/sake/index.json (hunk 1 above)
Comment (therealsakeswap) [reply spaces/sake/index.json:7]: yes，we needn't the  decimals() 
Is the decimals must needed in snap-shot?

Comment (therealsakeswap) [reply spaces/sake/index.json:7]: we use the decimals() default value(18)

Comment (therealsakeswap) [reply spaces/sake/index.json]: we have update the contract and add the function

Comment (therealsakeswap) [reply spaces/sake/index.json]: The decimals() is added.
Please review and check it.
Thanks

**Comment by therealsakeswap** [comment]
The decimals() is added.
Please review.Thanks
@e-asia

**Pull request closed without merging**